modules/random.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
       command: function (bot, msg) {
         if (msg.args.length > 2) {
           msg.args.shift()
-          return msg.args[Math.floor(Math.random() * msg.args.length)]
+          var randint = msg.args[Math.floor(Math.random() * msg.args.length)]
```

Comment:
```suggestion
          const randint = msg.args[Math.floor(Math.random() * msg.args.length)]
```

A minor bit of defensive programming: as you're never modifying this value, may as well change it to `const`.
(We're slowly removing all uses of `var` in favour of either `const` or `let`).